build_tools/scripts/generate_cmake_files.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Run all CMake file generators with proper output paths.
+
+set -euo pipefail
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+cd "${ROOT_DIR}"
+
+# Generates E2E test artifacts cmake files
+./build_tools/testing/generate_cmake_e2e_test_artifacts_suite.py \
+  --output_dir ./tests/e2e/test_artifacts
+
+# Generates E2E model test cmake files
+./build_tools/testing/generate_cmake_e2e_model_tests.py \
+  --output ./tests/e2e/models/generated_e2e_model_tests.cmake
```

Comment:
nit: these comments are redundant IMO (the script names say the same thing)

Also, are the output paths ever different? You could just hardcode the paths into the scripts themselves - less places to update / keep in sync

Reply:
Removed the comments.

IMO passing paths through command lines makes it easier to understand where those files are created. These paths are also relative to the git root. It's clearer to get git root and change the directory in shell script then in Python.

I updated the related READMEs so now all places point to this script to regenerate the cmake files. So this script is the only place to have these paths.